internal/providers/terraform/ibm/testdata/cloudant_test/cloudant_test.golden

```diff
@@ -0,0 +1,13 @@
+
+ Name                       Monthly Qty  Unit    Monthly Cost 
+                                                              
+ Cloudant - standard                                          
+ ├─ Monthly Reads                   100  second        $25.00 
+ ├─ Monthly Writes                   50  second        $25.00 
+ ├─ Monthly Global Queries            5  second        $25.00 
+ └─ Free Estimated Storage           20  GB             $0.00 
```

Comment:
Can you name this like `Storage free allowance (first 20GB)`

Reply:
In fact, can you also put another instance into the test with 40 GB of storage?

Reply:
yep